chucks-pick3-mobile/src/app/components/pick3-draw-date-card/pick3-draw-date-card.component.html

```diff
@@ -2,7 +2,7 @@
   <ion-row class="primary-view" [style.background]="'url(' + data.getBackgroundImage() + ')'">
     <ion-col>
       <span style="font-size: 1.5em;font-weight: bolder;">{{data.getDrawState()}} {{data.getTitle()}}</span><br><br>
-      <span style="font-size: 1.25em;font-weight: 500;">{{data.getDrawTime()}}</span><br><br>
+      <span style="font-size: 1.25em;font-weight: 500;">{{data.getDrawTimeAsString()}}</span><br><br>
```

Comment:
Nice! So the draw time is now being converted into a string instead of presenting the result asa number type?

Reply:
Yep!